test/metric_transaction_test.exs

```diff
@@ -93,6 +94,24 @@ defmodule MetricTransactionTest do
     assert TestHelper.find_metric(metrics, "External/allWeb")
   end
 
+  test "Request queueing transaction" do
+    conn =
+      conn(:get, "/foo/1")
+      |> Plug.Conn.put_req_header(
+        "x-request-start",
+        "t=#{:os.system_time(:microsecond) - 100_000}"
+      )
+
+    TestPlugApp.call(conn, [])
+
+    metrics = TestHelper.gather_harvest(Collector.Metric.Harvester)
+
+    assert [_, [1, time, time, time, time, 0]] =
```

Comment:
This test now fails because the metric isn't dispatched (I'm not actually sure, but would like to understand it better)

Reply:
Nevermind, it was operator error and works like I expected it to.